packages/open-payments/src/client/outgoing-payment.ts

```diff
@@ -100,6 +128,47 @@ export const createOutgoingPayment = async (
   }
 }
 
+export const listOutgoingPayments = async (
+  deps: BaseDeps,
+  getArgs: ListGetArgs,
+  validateOpenApiResponse: ResponseValidator<OutgoingPaymentPaginationResult>,
+  pagination?: PaginationArgs
+) => {
+  const { axiosInstance, logger } = deps
+  const { accessToken, paymentPointer } = getArgs
+  const url = `${paymentPointer}${getRSPath('/outgoing-payments')}`
```

Comment:
@wilsonianb

Reply:
yeah exactly what I was thinking. For collection requests/creates its pretty straightforward, but for the GETs, we could allow either full `url` to be passed in as an argument, or just an `id`

Reply:
For `GET`s and `/complete`s, I vote we at least do the full url.
I was going to suggest we even call the full url `id`, since that's the name of the field in responses. But we also refer to an `id` path parameter
https://docs.openpayments.guide/reference/get-incoming-payment

Reply:
If we just used `id` (no url or payment pointer url) for `GET`s and `/complete`, we could do things like
```
const payment = await client.incomingPayment.create({
  url: paymentPointerUrl
})
const currentPayment = await client.incomingPayment.get({
  id: payment.id
})
await client.incomingPayment.complete({
  id: payment.id
})
```